src/analysis/split.jl

```diff
@@ -6,18 +6,18 @@ Split doc string from given expression.
 function split_doc(ex::Expr)
     @switch ex begin
         @case Expr(:macrocall, &(GlobalRef(Core, Symbol("@doc"))), line, doc, expr) ||
-                Expr(:macrocall, &(Symbol("@doc")), line, doc, expr) ||
-                Expr(:macrocall, Expr(:., :Core, &(QuoteNode(Symbol("@doc")))), line, doc, expr)
-            return line, doc, expr
+              Expr(:macrocall, &(Symbol("@doc")), line, doc, expr) ||
+              Expr(:macrocall, Expr(:., :Core, &(QuoteNode(Symbol("@doc")))), line, doc, expr)
+        return line, doc, expr
 
         # quote
         #     @doc "" struct <name> end
         # end
         @case Expr(:block, ::LineNumberNode, stmt)
-            line, doc, expr = split_doc(stmt)
-            return line, doc, expr
+        line, doc, expr = split_doc(stmt)
+        return line, doc, expr
```

Comment:
I guess you run JuliaFormatter? It doesn't recognize MLStyle's match table unfortunately...